src/features/projectsV2/ProjectsMap/SingleProjectView.tsx

```diff
@@ -51,13 +59,33 @@ const SingleProjectView = ({ mapRef, setIsOnSampleMarker }: Props) => {
         setViewState,
         3500
       );
+    } else {
+      const pointPlantLocation =
+        selectedPlantLocation?.geometry?.type === 'Point'
+          ? selectedPlantLocation
+          : selectedSamplePlantLocation?.originalGeometry?.type === 'Point'
+          ? selectedSamplePlantLocation
+          : null;
+
+      if (pointPlantLocation) {
+        const [lon, lat] = pointPlantLocation.geometry.coordinates;
+        if (typeof lon === 'number' && typeof lat === 'number') {
+          // Zoom into single tree location (point)
+          zoomToLocation(setViewState, lon, lat, 20, 3500, mapRef);
+        }
+      }
     }
-  }, [selectedPlantLocation, requestedPlantLocation, router.isReady]);
+  }, [
+    selectedPlantLocation,
+    selectedSamplePlantLocation,
+    requestedPlantLocation,
+    router.isReady,
```

Comment:
What are the conditions under which we'd zoom to neither a polygon nor a point location? Handling those earlier will make it easier for typescript to exclude non-possible types.